docs/manual/resource-leak-checker.tex

```diff
@@ -632,16 +632,17 @@
 then verify the code using a method other than the Resource Leak Checker.
 
 
-%% TODO: Uncomment when the feature is ready to be publicized.
-% \sectionAndLabel{Resource Leak Checker Annotation Inference Algorithm}{resource-leak-checker-inference-algo}
-%
-% We are developing a special inference mechanism to infer annotations for
-% the Resource Leak Checker. By default, this mechanism operates when the
-% -Ainfer flag is enabled. To utilize the whole-program inference mechanism outlined in
-% section \ref{whole-program-inference} in addition to this mechanism for the Resource
-% Leak Checker and its sub-checkers, you should provide the \<-AenableWpiForRlc> flag.
-%
-% The paper \href={https://arxiv.org/pdf/2306.11953.pdf}{Automatic Inference of Resource Leak Specifications}, published at OOPSLA 2023, gives more details on the inference algorithm. Currently, only the first phase of the algorithm described in the paper is available, and the complete implementation will be available soon.
+\sectionAndLabel{Resource Leak Checker Annotation Inference Algorithm}{resource-leak-checker-inference-algo}
+
+We have developed a specialized algorithm to infer annotations for
+the Resource Leak Checker. This algorithm runs by default when the
+\<-Ainfer> flag is enabled. To utilize the whole-program inference mechanism outlined in
```

Comment:
"This algorithm runs by default when the \<-Ainfer> flag is enabled." is a bit misleading, since 1) that's only true if you run the RLC and 2) it doesn't make it clear that this algorithm runs _instead of_ rather than _in addition to_ the standard WPI algorithm. I'd clarify this sentence to the following:
"The RLC uses this algorithm by default when the \<-Ainfer> flag is enabled, instead of the standard whole-program-inference (WPI) algorithm described in section \ref{whole-program-inference}."

Then, you can cut most of the first part of the next sentence, so that it becomes:
"To use WPI in addition to this mechanism ..."

Reply:
Addressed in c4af094